.kokoro/bigtable-conformance.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
This file shouldn't be in the .kokoro folder, because it's not being run by Kokoro. I think it should go on the Google.Cloud.BigTable.V2.ConformanceTests folder.